projects/packages/forms/src/contact-form/css/file-field.css

```diff
@@ -124,3 +123,21 @@
 	color:  var( --jetpack--contact-form--button-primary--text-color, #FFF );
 	margin-right: 10px;
 }
+
+@keyframes showFileField {
```

Comment:
Keyframe names are in global scope just like CSS selectors, let's prefix them with `jetpack_`